src/pages/sns/faq.tsx

```diff
@@ -741,6 +809,48 @@ function SnsFaqPage() {
                 </Link>
               </p>
             </Faq>
+            
+            <Faq title="What is voting power and how do I get it?">
+              SNS DAOs have a stake-based governance system. This means the more
+              tokens participants stake in their{" "}
+              <Link href="https://wiki.internetcomputer.org/wiki/Neurons_101">
+                neurons
+              </Link>, the more voting power their neurons have. In addition to
+              the stake, a neuron's voting power
+              is dependent on the{" "}
+              <Link href="https://wiki.internetcomputer.org/wiki/Neurons_101">
+                  dissolve delay
+              </Link> bonus and age bonus. Each SNS DAO
+              decides these bonuses in its parameters.
+            </Faq>
+            
+            <Faq title="What is neuron following?">
+              <p>
+                Following neurons is a way to delegate votes both on the NNS and in SNSs.
+                Each SNS neuron can either manually vote on proposals or
+                follow another neuron’s voting decisions. Neurons are followed based on
+                various topics, such as making SNS treasury transfers or upgrading the dapp.
+                Neurons can be set to follow other neurons on specific topics or to follow
+                other neurons on ‘All Topics’ – a catch-all which is applied for the proposal
+                without any specific following choice. In both cases, you must trust that
+                the neurons you’re following will make decisions that align with your views.
```

Comment:
```suggestion
                without any specific following choice.
```
(would remove it here as it is now in the next point)